src/common-rest-api/src/main/java/org/niis/xroad/restapi/auth/securityconfigurer/StaticAssetsWebSecurityConfig.java

```diff
@@ -50,6 +50,14 @@ protected void configure(HttpSecurity http) throws Exception {
                         "/js/**",
                         "/fonts/**")
                 .and()
+            .headers()
+                    .contentSecurityPolicy(
+                            "default-src 'self' 'unsafe-inline'  ;"
+                            + "script-src 'self' 'unsafe-inline' 'unsafe-eval' ;"
```

Comment:
Maybe we don't need all unsafe-xxx definitions? Implementation would in principle be a bit more secure if we can avoid some. I will put more in the Jira ticket.

Reply:
Fix proposal: abf32ba

Reply:
Updated set of CSP -headers are added now  both to SS (8db52e5) and (at last) to CS (8b4f594).
Also STS header for CS has been added now (bc80ba7).